tests/integration/local/common_utils.py

```diff
@@ -28,8 +28,13 @@ def wait_for_local_process(process, port, collect_output=False) -> str:
         if "Address already in use" in line_as_str:
             LOG.info(f"Attempted to start port on {port} but it is already in use, restarting on a new port.")
             raise InvalidAddressException()
-        if "Press CTRL+C to quit" in line_as_str or "Terraform Support beta feature is not enabled." in line_as_str:
+        if (
+            "Press CTRL+C to quit" in line_as_str
+            or "Terraform Support beta feature is not enabled." in line_as_str
+            or "Error: " in line_as_str
```

Comment:
Is there a case where there is any other output after an `Error: ` occurs in the output? In theory, this means that an exception was raised before we started the Flask application so the command execution stops there.

Reply:
I was thinking if some testing lambda function writes `Error:` and it was expected.

Reply:
Hmm, thats a good point. Lambda invocations would have happened after this loop (since the `CTRL+C` check breaks once the Flask app is ready), but there would be other output that happens before hand that just happens to contain `Error:`. I'll see what I can do to make this check more specific, or perhaps changing the authorizer specific tests to expect the command to not setup correctly, instead of reusing this method.

Reply:
you are correct, I missed the point that the lambda invocation done after that. It is my bad. So, in this case I think you can ignore my comment.